Makefile

```diff
@@ -1,36 +1,41 @@
 
 # Default configuration
-SHELL     := /bin/bash
-PYTHON    := python3
-PIP       := ${PYTHON} -m pip
-LINTER    := pycodestyle
-FORMATTER := black
-MAKE_TAG  := --no-print-directory
+SHELL       := /bin/bash
+PYTHON      := python3
+PIP         := ${PYTHON} -m pip
+LINTER      := pycodestyle
+FORMATTER   := black
+MAKE_TAG    := --no-print-directory
 
 # Virtual env
-VENV_NAME = venv
+VENV_NAME   := venv
+VENV_PYTHON := ${VENV_NAME}/bin/${PYTHON}
 
 
 .PHONY: echo-cyan, echo-purple, echo-green, requirements, lint, format, clean
 
 
 ## Create virtualenv
 venv:
-	@$(MAKE) $(MAKE_TAG) echo-cyan msg="Creating and seting up a virtualenv..."
+	@$(MAKE) $(MAKE_TAG) echo-cyan msg="Creating and setting up a virtualenv..."
 	@$(PYTHON) -m venv $(VENV_NAME)
-	@. $(VENV_NAME)/bin/activate; \
-	$(PIP) install --upgrade pip; \
-	$(PIP) install -r requirements.txt
-	@$(MAKE) $(MAKE_TAG) echo-green msg="Virtual created environment successfully ! ✨"
-	@echo
-	@echo "To activate it, please run 'source venv/bin/activate'"
+	@. $(VENV_NAME)/bin/activate;
+	@$(MAKE) $(MAKE_TAG) echo-green msg="Virtual environment created successfully ! ✨"
 
 ## Update requirements.txt file
 requirements: venv
+	$(PIP) install --upgrade pip
+	$(PIP) install -r requirements.txt
 	@rm requirements.txt 2> /dev/null
-	@. venv/bin/activate; \
 	$(PIP) freeze > requirements.txt
-	@echo "$@.txt updated! ✨\n"
+	@$(MAKE) $(MAKE_TAG) echo-green msg="\n$@.txt updated! ✨\n"
+
+migration:
+	@$(VENV_PYTHON) manage.py makemigrations
+	@$(VENV_PYTHON) manage.py migrate
+
+serve-back:
```

Comment:
I don't think we would need a target like that.

I won't use it because I prefer to run `python3 manage.py runserver` hehe

Reply:
It's mainly for my own. I hate writing `python3 manage.py runserver` 😂

Reply:
hahaha okay that's fine. You can then make this target `.PHONY`

Reply:
Oh yes, I forgot it